pkg/private/pkg_files.bzl

```diff
@@ -515,7 +518,7 @@ def _encode_manifest_entry(dest, df, use_short_path):
         src = None
     return json.encode([
         entry_type,
-        dest.strip("/"),
+        dest.rstrip("/"), # do not remove leading /, since we meant(!?) it
```

Comment:
are there any of these hacks like this that we can just fix in the vehicle repo?

would ____love____ to drop these changes entirely, or at least move closer to vanilla rules_pkg in some way

Reply:
There isn't much that can be done from the vehicle repo. The issue is the unmodified rules_pkg always uses a relative path and will proactively strip leading / from paths. If we don't want to keep this behavior we could remove a lot of the hacks. The side effect is when unpacking tarballs, we'd need to be sure the current directory matches the tarball root directory or else copy the uncompressed files to the correct location.

Reply:
have you identified which users in the vehicle repo are relying on this behavior?  that may be a good start.

Reply:
I don't know who is using the binary tarballs, but the number of builds that have package_dir = "/something" is quite a lot, 70+.